### PR TITLE
Meta: Link to changelog and Linus' Serenity VNC

### PR DESCRIPTION
**Pull request by BenWiederhake**

### CODE REVIEW

File: Meta/Websites/serenityos.org/index.html [hunk 1]
```diff
@@ -52,6 +52,8 @@ <h1>SerenityOS</h1>
     <li><a href="happy/2nd/">Happy 2nd birthday! SerenityOS: The second year</a></li>
     <li><a href="happy/1st/">Happy 1st birthday! SerenityOS: From zero to HTML in a year</a></li>
     <li><a href="https://happy-serenityos.linus.dev/">Linus's ":^)" tracker</a></li>
+    <li><a href="https://serenity.linus.dev/">VNC-access to Serenity running on Linus's hardware</a></li>
+    <li><a href="https://changelog.serenity/">Lubrsi's commit overview, grouped by month and category</a></li>
```
Comment (supercomputer7) [inline Meta/Websites/serenityos.org/index.html:56]: I think you meant `https://changelog.serenityos.org`?

File: Meta/Websites/serenityos.org/index.html [hunk 2]
```diff
@@ -52,6 +52,8 @@ <h1>SerenityOS</h1>
     <li><a href="happy/2nd/">Happy 2nd birthday! SerenityOS: The second year</a></li>
     <li><a href="happy/1st/">Happy 1st birthday! SerenityOS: From zero to HTML in a year</a></li>
     <li><a href="https://happy-serenityos.linus.dev/">Linus's ":^)" tracker</a></li>
+    <li><a href="https://serenity.linus.dev/">VNC-access to Serenity running on Linus's hardware</a></li>
```
Comment (linusg) [inline Meta/Websites/serenityos.org/index.html:55]: Please don't. I don't want to give any guarantees about uptime, and too many people using it makes the single instance unusable really quickly (even from a shared keyboard/mouse perspective). It's ok when people find it from Discord or the original announcement though, but I don't want it linked :)

**Review by linusg — CHANGES_REQUESTED** [review]

File: Meta/Websites/serenityos.org/index.html (hunk 1 above)
Comment (BenWiederhake) [reply Meta/Websites/serenityos.org/index.html:56]: Derp >_<

File: Meta/Websites/serenityos.org/index.html (hunk 2 above)
Comment (BenWiederhake) [reply Meta/Websites/serenityos.org/index.html:55]: Ah, sorry, then I misunderstood you. Sure!

**Comment by BenWiederhake** [comment]
Changes since initial push:
- Removed the reverence to the VNC instance
- Fixed the URL (How the hell did I manage to get that wrong in the first place!)

**Comment by IdanHo** [comment]
super nitpick: the commit mentions sub-projects in the plural, while only one is added

**Comment by BenWiederhake** [comment]
Changes since last push:
- Changed the commit title for super-hardcore nitpick

**Review by IdanHo — APPROVED** [review]
Thanks!

**Pull request merged**